servicedirectory/snippets_test.py

```diff
@@ -17,24 +17,24 @@
 from os import environ
 import uuid
 
+from _pytest.capture import CaptureFixture
```

Comment:
Prefer testing a response instead of captured output. Functions that just print a message can return a status indicating success or failure.